src/modules/main.haml

```diff
@@ -161,17 +161,29 @@
                                 %td The contribution this author or contributor made to the map.
                                 %td
                                     %span.label.label-primary String
+                            %tr
+                                %td
+                                    %code uuid
+                                %td UUID used to define a player.
+                                %td
+                                    %span.label.label-primary String
+
+                %br
+                A players UUID is a unique code to each player that is used to keep track of each player even if they change their name. You can check player UUID's on
+                = succeed "." do
+                    %a{:href => "http://mcuuid.net/"} mcuuid.net
+                %br
```

Comment:
You should move this whole block (starting at the first `%br`) down into the markdown parser, removing the `succeed do` nonsense.

Also document the fact that if an author or contributor is defined without a uuid that player will not get any mapmaker perks on the map.